gitkit/gitkit.go

```diff
@@ -446,3 +450,38 @@ func extractRemoteIP(req *http.Request) string {
 	}
 	return host
 }
+
+func retrieveProjectConfig(apiClient *APIClient, conf *Config) error {
```

Comment:
I think you should allow the developer to overwrite configurations per app. The current solution is doing the opposite.